corehq/apps/analytics/static/analytics/js/deprecated.js

```diff
@@ -0,0 +1,133 @@
+/* globals _, _kmq */
```

Comment:
As things get deprecated, it'd be good to update the [analytics README](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/analytics/README.md).

Reply:
yes, absolutely!